docs/gettingstarted/index.rst

```diff
@@ -210,39 +210,31 @@ Getting started with Flocker
       You should have the Flocker client installed on your laptop and flocker-node installed on some servers: either VMs on your laptop, or real instances on cloud infrastructure.
       Now you can try our simple tutorial: a Python web application and a Redis server.
 
-      .. code-block:: console
+      .. container:: hidden
 
-         you@laptop:~$ git clone https://github.com/clusterhq/flocker-quickstart
-         you@laptop:~$ cd flocker-quickstart
+         .. Create the files to be downloaded with curl, but don't show download links for them
 
-      fig.yml
-      -------
+         :download:`fig.yml`
+         :download:`deployment-node1.yml`
+         :download:`deployment-node2.yml`
 
-      .. code-block:: yaml
+      .. version-code-block:: console
 
-         web:
-           image: clusterhq/flask
-           links:
-            - "redis:redis"
-           ports:
-            - "80:80"
-         redis:
-           image: redis
-           ports:
-            - "6379:6379"
-           volumes: ["/data"]
+         you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/fig.yml
+         you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/deployment-node1.yml
+         you@laptop:~$ curl -O https://docs.clusterhq.com/en/|latest-installable|/_downloads/deployment-node2.yml
```

Comment:
I think replacing one `git clone ...` command with three `curl ...` commands will probably annoy some people (even though this is just copy/paste stuff).

I think you could fix this with shell shenanigans, eg `curl ... /_downloads/{fig,deployment-node1,deployment-node2}.yml`.  Or we could host a tar archive and `curl ... | tar xf -` - though I don't really want to spend time on the setup that would require.  Maybe just checking an uncompressed tar archive into the repo would be an okay interim solution (with an issue filed for follow-up to replace it with unarchived files and a doc generation step that creates the tar archive).

Thoughts?